swarm/src/connection/pool.rs

```diff
@@ -783,13 +791,14 @@ where
                         )
                         .boxed(),
                     );
-
+                    let established_in = Instant::now() - accepted_at;
```

Comment:
Raising my previous comment here once more:

```suggestion
                    let established_in = accepted_at.elapsed();
```

Let's use [`elabsed`](https://doc.rust-lang.org/nightly/std/time/struct.Instant.html#method.elapsed) here.

https://github.com/libp2p/rust-libp2p/pull/3134#discussion_r1041329359